lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1473,7 +1473,7 @@ sub _asset_find {
     # add undef to parents so that we check regular assets too
     for my $parent (@$parents, undef) {
         my $fname = $parent ? sprintf("%08d-%s", $parent, $name) : $name;
-        return $fname if (locate_asset($type, $fname, mustexist => 1));
+        return $fname if OpenQA::Utils::locate_asset($type, $fname, mustexist => 1);
```

Comment:
You don't need to change it for the sake of mocking. You can do `MockModule->('OpenQA::Schema::Result::Jobs')`.

Reply:
Thanks for the hint, I'll try it.

Reply:
works, updated the PR